<launch>
    <node name="trajectory_final" pkg="m100_trajectory" type="trajectory_final" output="screen"/>
	<param name="verbose_traj" type="bool" value="true" />
	<param name="mocap_topic" type="string" value="/mavros/mocap_gps/pose" />
    	<param name="Llidar_topic" type="string" value="/teraranger_evo_l" />
	<param name="Clidar_topic" type="string" value="/teraranger_evo_c" />
    	<!--param name="Clidar_topic" type="string" value="/teraranger_duo/infrared" /-->
    	<param name="Rlidar_topic" type="string" value="/teraranger_evo_r" />
	<param name="use_sonar" type="bool" value="false" />
	<param name="sonar_topic" type="string" value="/teraranger_duo/ultrasound" />

    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />
</launch>
